src/scheduler/MiscJSDoc.ts

```diff
@@ -1,85 +0,0 @@
-import {Subscription} from '../Subscription';
-import {Scheduler} from '../Scheduler';
-
-/**
- * A unit of work to be executed in a {@link Scheduler}. An action is typically
- * created from within a Scheduler and an RxJS user does not need to concern
- * themselves about creating and manipulating an Action.
- *
- * ```ts
- * interface Action extends Subscription {
- *   work: (state?: any) => void|Subscription;
- *   state?: any;
- *   delay?: number;
- *   schedule(state?: any, delay?: number): void;
- *   execute(): void;
- *   scheduler: Scheduler;
- *   error: any;
- * }
- * ```
- *
- * @interface
- * @name Action
- * @noimport true
- */
```

Comment:
Yeah it's correct. I just commented so others can ignore these changes, they're already verified and LGTM. I didn't review carefully the whole PR so I didn't give my LGTM yet